src/foam/dao/EasyDAO.js

```diff
@@ -86,7 +92,9 @@ foam.CLASS({
       /** The developer-friendly name for this EasyDAO. */
       class: 'String',
       name: 'name',
-      factory: function() { return this.of.id; }
+      factory: function() {
+ return this.of.id;
```

Comment:
indentation is broken.

Reply:
You are looking at an outdated version of the code. I have fixed this now.